src/modules/gpu.rs

```diff
@@ -110,6 +110,13 @@ pub fn get_gpus(config: &Configuration) -> Result<Vec<GPUInfo>, ModuleError> {
         Err(e) => return Err(e)
     }
 
+    if gpus.is_empty() {
+        match fill_from_glxinfo(&mut gpus) {
```

Comment:
No. This fucks performance, not doing it. Find an alternative detection.

Reply:
Fuck what? It fucks only if `fill_from_pcisysfile` fails, so it only fucks ARM devices.

Reply:
Yes, but I refuse to believe there's not better ways of detecting the name from arm than using glxinfo.